utils/submission.js

```diff
@@ -700,6 +710,9 @@ const getVersionNumberFromGitHubCommit = async (sha, path, token) => {
     }
 }
 
+// const { processPromisResults } = require('./promis');
+// processPromisResults('eyjekCU9H4V4crXpcRp95HINkXI2');
```

Comment:
These 2 lines may not be needed.

Reply:
thought I deleted those, thanks for catching